engine/src/main/scala/cromwell/engine/backend/dummy/DummyInitializationActor.scala

```diff
@@ -0,0 +1,50 @@
+package cromwell.engine.backend.dummy
+
+import akka.actor.Props
+import cromwell.backend.BackendLifecycleActor.{BackendWorkflowAbortSucceededResponse, WorkflowAbortResponse}
+import cromwell.backend.{BackendConfigurationDescriptor, BackendWorkflowDescriptor, BackendWorkflowInitializationActor}
+import cromwell.core._
+import wdl4s.Call
+
+import scala.concurrent.Future
+import scalaz.NonEmptyList
+
+object DummyInitializationActor {
+  def props(workflowDescriptor: BackendWorkflowDescriptor, calls: Seq[Call], configDesc: BackendConfigurationDescriptor): Props =
+    Props(new DummyInitializationActor(workflowDescriptor, calls, configDesc))
+}
+
+// The sequence of calls that this receives are the ones this backend is expected to run, and it should validate only those
+class DummyInitializationActor(override val workflowDescriptor: BackendWorkflowDescriptor,
+                               override val calls: Seq[Call],
+                               override val configurationDescriptor: BackendConfigurationDescriptor) extends  BackendWorkflowInitializationActor{
+
+  private val tag = s"[${self.path.name}]"
+
+  override def validateRuntimeAttributes(runtimeAttributes: EvaluatedRuntimeAttributes): Future[scalaz.Validation[NonEmptyList[String], Unit]] =
+  Future.successful(scalaz.Success(Unit))
```

Comment:
If this isn't validating anything, we don't need a `DummyInitializationActor`. `InitializationActor`s should be optional